FinniversKit/Sources/Fullscreen/FullscreenGallery/FullscreenGalleryViewController.swift

```diff
@@ -268,7 +268,7 @@ extension FullscreenGalleryViewController: FullscreenImageViewControllerDataSour
         }
 
         let url = viewModel.imageUrls[vc.imageIndex]
-        let imageWidth = min(view.bounds.width, view.bounds.height)
+        let imageWidth = CGFloat(3840)
```

Comment:
What's stopping us from just using the original image size in fullscreen? Is the size increase so large for some images that it's an issue for us?

Reply:
I went for that first, but since there is no ready method for fetching the original CDN image afaik and that the urls come in two different formats (CDN and AppsGw) I pondered that perhaps this change would come with the same result without the hassle of the contingencies.